### PR TITLE
#130 Quick hack for compatibility with Docker 1.13.0

### PR DESCRIPTION
**Pull request by dryewo**
If config.mediaType is equal to "application/json" in the manifest, set it to "application/vnd.docker.container.image.v1+json".
Addresses #130.

### CODE REVIEW

**Comment by dryewo** [comment]
👍

File: Dockerfile [hunk 1]
```diff
@@ -1,4 +1,4 @@
-FROM registry.opensource.zalan.do/stups/openjdk:8-42
+FROM registry.opensource.zalan.do/stups/openjdk:8-cd27
```
Comment (hjacobs) [inline Dockerfile]: why not just `:latest` ("fake" tag which works great for base images)

Comment (dryewo) [reply Dockerfile]: ok, if you insist

**Comment by dryewo** [comment]
👍

**Comment by hjacobs** [comment]
:+1:

**Comment by Raffo** [comment]
This should not be needed anymore: https://github.com/docker/docker/pull/30262

**Pull request merged**